common/djangoapps/pipeline_mako/__init__.py

```diff
@@ -14,7 +14,7 @@ def compressed_css(package_name, raw=False):
 
     package = packager.package_for('css', package_name)
 
-    if settings.PIPELINE:
+    if settings.PIPELINE_ENABLED:
```

Comment:
This is linked to an upgrade of django_pipeline, but should be independent of django 1.5